jobs/JGLOBAL_LAND_ANALYSIS_FINALIZE

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+export WIPE_DATA="NO"
+export DATA=${DATA:-${DATAROOT}/${RUN}landanl_${cyc}}
+source "${HOMEgfs}/ush/jjob_header.sh" -e "landanlfinal" -c "base landanl landanlfinal"
+
+##############################################
+# Set variables used in the script
+##############################################
+export CDATE=${CDATE:-${PDY}${cyc}}
+export CDUMP=${CDUMP:-${RUN:-"gfs"}}
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+
+GDATE=$(date +%Y%m%d%H -d "${CDATE:0:8} ${CDATE:8:2} - ${assim_freq} hours")
+export GDATE
+gPDY=${GDATE:0:8}
+export gcyc=${GDATE:8:2}
+export GDUMP=${GDUMP:-"gdas"}
+
+export OPREFIX="${CDUMP}.t${cyc}z."
+export GPREFIX="${GDUMP}.t${gcyc}z."
+export APREFIX="${CDUMP}.t${cyc}z."
```

Comment:
Since these are not needed at the j-job level, they should be defined lower where they are needed, such as in the `__init__` of `LandAnalysis` class.

Reply:
Thanks @aerorahul The script `jobs/JGLOBAL_LAND_ANALYSIS_FINALIZE` was created by following the script `jobs/JGLOBAL_AERO_ANALYSIS_FINALIZE` for the aerosol DA. 
@CoryMartin-NOAA What do you suggest and can I simply remove these? Thanks.

Reply:
I defer to @aerorahul on this, but if this is the decision made then other files such as my aero j-jobs also will need this removed in a future PR.

Reply:
This is done following aerosol DA.

Reply:
I think I removed it in the aerosol tasks.

Reply:
@aerorahul Yes, I removed them too following the aerosol task.